docs/podman-volume-create.1.md

```diff
@@ -39,6 +45,8 @@ $ podman volume create myvol
 $ podman volume create
 
 $ podman volume create --label foo=bar myvol
+
+# podman volume create --opt type=nfs --opt o=addr=192.168.0.2,rw --opt device=:/nfsshare  mynfsvol
```

Comment:
Should this be `device=:/nfsshare` or `device=/nfsshare` (no colon)?

Reply:
Both are supported.